app/views/PrintSummaryPage.scala.html

```diff
@@ -63,29 +140,43 @@ <h1 class="govuk-heading-xl">
         )
     ))
 
-    @upfrontPaymentAmount.map { upfrontAmount =>
-
-        <h2 class="govuk-heading-l">
-            @{
-                Messages.PaymentSchedule.`Upfront payment`.show
-            }
-        </h2>
+    <h2 class="govuk-heading-m">
+    @{
+        Messages.PaymentSchedule.`Upfront payment`.show
+    }
+    </h2>
 
     @govukSummaryList(SummaryList(
         rows = Seq(
             SummaryListRow(
                 key = Key(
-                    content = HtmlContent(Html(Messages.PaymentSchedule.`Taken within 10 working days`.show)),
+                    content = HtmlContent(Html(Messages.UpfrontPayment.`Can you make an upfront payment?`.show)),
                     classes = "govuk-!-width-one-half"
                 ),
                 value = Value(
-                    content = Text(upfrontAmount.formatInPounds)
+                    content = Text(upfrontPaymentAmount.fold(Messages.UpfrontPayment.`No`)(_ => Messages.UpfrontPayment.`Yes`).show)
                 ),
                 actions = None
             )
-
-        )
+        ).toList.++(maybeUpfrontPaymentAmount)
     ))
+
+    <h2 class="govuk-heading-m">
+    @{
+        Messages.PaymentSchedule.`Monthly payments`.show
     }
-}
+    </h2>
+
+    @govukSummaryList(SummaryList(
+        rows = monthlyPaymentRows
+    ))
+
+    <p class="govuk-body">
```

Comment:
we should prevent users with no JavaScript enabled from seeing this entire paragraph as it will not work for those users - we can add a custom CSS class to do that, e.g. `js-visible` 
```
<p class="govuk-body js-visible">
```

and add the necessary SASS to the application.scss file e.g. 
```
body:not(.js-enabled) .js-visible {
  display: none;
}
```
this makes use of the govuk-frontend checks that add `js-enabled` to the body when JS is detected as supported

Reply:
Think I've added what's required @nubz